### ISSUE TITLE
[Feature] Column description and data type hints when hovering tiles

### ISSUE THREAD

**Issue by JoeMark17** [Cedar-Cat-Studios/dbt_query_game#5]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

Currently it is quite difficult for players to understand what the column type and descriptions of the columns are when playing the game. It would be a good feature to include a "hint" feature that displays the column description and datatype when the player is hovering their mouse over the column tile. 

This will ensure players are aware of what exactly the column metadata details are without needing to either run a query (and waste an attempt) or come to this repo to view the [ERD](https://drawsql.app/teams/cedar-cat-studios/diagrams/jaffle-shop).

### Anything else?

Not sure how this will look in a small window that exists when hovering over the tiles, but it will be difficult to play the line between helpful and too noisy.

**Comment by JoeMark17** [comment]
The functionality for this tool is currently workin on the development build of the game. It is highly likely that the next release of the game will have the helper icon when hovering over fields.

The current debate I am having is if the helper should appear immediately, or only after the player has hovered for a second or so. 🤔

**Comment by JoeMark17** [comment]
After play testing this a bit myself, it seems to work really well when the helper pops up immediately. If there is the delay then it feels a bit laggy and unresponsive. As a result, I am opting to go the route of immediately showing the helper.

Additionally, I am opting to not include the helper on the table tiles themselves. I feel it will be duplicative and unnecessary. I also was able to get player feedback from @JessicaAKelley that this would indeed be duplicative and is not needed. For example, each table tile would just be, "Table containing all X records." 

Finally, I have been able to implement this feature in the dev build of `v0.2.3`! It will require regenerating all the table tiles again, but I don't think that will be too much of a set back as I was wanting to enlarge the tables since they seem a bit small when playing.

You will be able to expect to see this feature implemented in the next release!
